spec/models/queued_video_spec.rb

```diff
@@ -30,11 +31,39 @@
       video = Fabricate(:video, reviews: [review])
       queued_video = Fabricate(:queued_video, user: user, video: video)
 
-      expect(queued_video.review_rating).to eq("")
+      expect(queued_video.rating).to eq("")
     end
   end
 
-  describe 'video_title' do
+  describe 'rating=' do
```

Comment:
You should use RSpecs convention to describing methods. `.` is prepended for class methods and `#` for instance level methods. Read more here - http://betterspecs.org/#describe